src/Routes.tsx

```diff
@@ -8,6 +8,7 @@ import Pricing from './pages/Pricing';
 import PricingForm from './components/PricingForm';
 import AdminLogin from './pages/Organization/AdminLogin';
 import Perfomancetraineetable from './components/TraineePerformance';
+import Noredirect from './pages/Noredirect';
 
 function MainRoutes() {
   return (
```

Comment:
We are missing the route to go to the `pricing` page.

Please fix the issue so that we can test the feature